### DISCUSSION TITLE
主题不支持本地文章封面？

### DISCUSSION THREAD

**Discussion by AGEN233** [valor-x/hexo-theme-solitude#302] [Q&A]
在cover填入图片路径之后，仍然会识别成ERROR，随之显示post_bg entered error与404图片

**Comment by everfu** [comment]
路径错了吧
图片需要在 source 文件夹下，路径为 /path

**Reply by AGEN233** [reply]
必须source/path吗，我是放在了source /_posts/文章名/path下

**Reply by AGEN233** [reply]
cover: /top.webp
浏览器生成的是
 `<a href="/posts/66b5bdfd/" title="“送去大学的外卖基本没有差评”" data-pjax-state=""><img class="post_bg entered error" src="/img/error_load.avif" data-lazy-src="/top.webp" alt="“送去大学的外卖基本没有差评”" data-ll-status="error"></a>`

**Reply by everfu** [reply]
没试过，好像不回去识别/posts/路径下的图片

**Reply by everfu** [reply]
试过了，posts文件夹内的图片不会识别

**Reply by AGEN233** [reply]
大佬有计划修复嘛，这种方式导入方式貌似还挺常见的，至少在butterfly系主题里面经常这样（

**Reply by everfu** [reply]
这和主题有啥关系

**Reply by AGEN233** [reply]
就是这里才奇怪（
我这边是把butterfly原版或者鱼的主题挪过去就行，目前试下来这边butterfly系主题全部支持，但是solitude不行。
butterfly系主题生成的是类似于这样的地址
`<img class="post_bg entered loaded" src="/posts/b8ae5ec7/top.webp" onerror="this.onerror=null,this.src=&quot;/img/404.jpg&quot;" data-lazy-src="/posts/b8ae5ec7/top.webp" alt="imgtp图床寄了" style="pointer-events: none" draggable="false" data-ll-status="loaded">`

**Reply by everfu** [reply]
你试试最新的 dev 分支，我刚才交了

**Reply by everfu** [reply]
https://github.com/everfu/hexo-theme-solitude/commit/c9cd2e4ec8b399d0d05c0ad0fb29c07883f7f4df
